listener/events.go

```diff
@@ -65,7 +65,7 @@ func HandleDelete(event mqueue.PlatformEvent) error {
 	defer utils.ObserveSecondsSince(tStart, messageHandlingDuration.WithLabelValues(EventDelete))
 	// TODO: Do we need locking here ?
 	err := database.OnConflictUpdate(database.Db, "inventory_id", "when_deleted").
-		Save(models.DeletedSystem{
+		Create(models.DeletedSystem{
```

Comment:
Was the "Save" method removed from new version of Gorm?

Reply:
Save method will UPDATE, we need to INSERT.